navis/interfaces/h01.py

```diff
@@ -0,0 +1,40 @@
+from functools import lru_cache
+from . import cave_utils
+
+@lru_cache(None)
```

Comment:
`cave_utils.get_cave_client` is already cached, so this decorator is superfluous.